tests/qchem/of_tests/test_convert.py

```diff
@@ -1966,7 +1955,8 @@ def test_rcisd_state_energy(molecule, basis, symm, charge, spin, tol):
 
     h_ferm = qchem.fermionic_observable(core_constant, one_mo, two_mo)
     H = qchem.qubit_observable(h_ferm)
-    energy_pl = np.conj(wf_cisd.T).dot(qml.matrix(H).dot(wf_cisd))
+    H_mat = H.sparse_matrix().toarray()
```

Comment:
Do you have a rough timing comparison for these tests with and without `active_new_opmath`?

Reply:
Under new operation arithmetic using `sparse_matrix` gives you about 10x better times but using `matrix` the degradation was about 120x.

Reply:
Note that this is because we need to update the `matrix` method of `Sum` which currently does not use the pauli representation but should. I am going to push a fix shortly. Good catch!

Reply:
See https://github.com/PennyLaneAI/pennylane/pull/5392 though sadly this is currently blocked by identity matrices 😢 there should be a fix though

Reply:
No worries, that's why I shifted to `sparse_matrix` to not get blocked by this.